tests/test-firebase-test-lab/build.gradle

```diff
@@ -3,6 +3,7 @@ buildscript {
     ext.kotlin_version = System.getenv("KOTLIN_VERSION")
     repositories {
         google()
+        gradlePluginPortal()
```

Comment:
Why do you need this one ?

Reply:
I had issues to downloaded the news Gradle version and this line seemed to fix my issue. But I have re-tested without and had no problem, so I have removed it.